spec/models/course_option_spec.rb

```diff
@@ -21,11 +21,67 @@
   end
 
   describe '.selectable' do
+    subject(:course_option) { create(:course_option) }
+
     it 'returns only course options where invalidated_by_find is false' do
       expected_course_option = create(:course_option, invalidated_by_find: false)
       create(:course_option, invalidated_by_find: true)
 
       expect(CourseOption.selectable).to match_array [expected_course_option]
     end
   end
+
+  describe '#course_not_available?' do
+    it 'returns true if course is not exposed in find' do
+      option = build_stubbed(
+        :course_option,
+        course: build_stubbed(:course, exposed_in_find: false),
+      )
+      expect(option.course_not_available?).to be true
+    end
+
+    it 'returns false if course is exposed in find' do
+      option = build_stubbed(
+        :course_option,
+        course: build_stubbed(:course, exposed_in_find: true),
+      )
+      expect(option.course_not_available?).to be false
+    end
+  end
+
+  describe '#course_full?' do
+    let(:course) { course_option_under_test.course }
+
+    context 'object under test has vacancies' do
```

Comment:
*flashes back to the one time he asked "why do all our tests have this mysterious `sut` variable" and the QA engineer shouted at him "SYSTEM UNDER TEST"*

Reply:
```suggestion
    context 'course option has vacancies' do
```

Maybe? (minor/stylistic suggestion, feel free to ignore)